### DISCUSSION TITLE
Should I upgrade pip whenever the console tells me there's a new update version?

### DISCUSSION THREAD

**Discussion by TKS4** [vladmandic/sdnext#876] [Q&A] [answered]
So yeah, periodically, going back to even 1111's repo, I would get a message sometimes telling me a new pip version was available. Is it safe to update pip generally, or should I just ignore such message informing me I should update? I've updated it in the past and noticed no problems, but was wondering if pip itself is already established enough to not warrant caution against updating to new versions?

**Comment by vladmandic** [comment, marked as answer]
its safe, but typically not needed. never seen a case where pip upgrade broke things or pip was too low to start with as pip operations used by app are pretty basic.
